src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/prison/UserControllerIntTest.kt

```diff
@@ -177,35 +177,39 @@ class UserControllerIntTest : IntegrationTestBase() {
       hmppsAuthMockServer.stubForNewToken()
       externalUsersApiMockServer.stubValidEmailDomain()
 
-      val prisonUserDetails = webTestClient.post().uri("/prisonusers")
+      webTestClient.post().uri("/prisonusers")
         .headers(setAuthorisation(roles = listOf("ROLE_CREATE_USER")))
         .body(
           fromValue(
             mapOf(
-              "username" to "TEST1",
-              "email" to "test@gov.uk",
-              "firstName" to "Test",
+              "username" to "TEST_ADM",
+              "email" to "testadm@gov.uk",
+              "firstName" to "Testadm",
               "lastName" to "User",
               "userType" to "DPS_ADM",
             ),
           ),
         )
         .exchange()
         .expectStatus().isCreated
-        .expectBody(NewPrisonUserDto::class.java)
```

Comment:
Even though this check is for NewPrisonUserDto - it wasn't an exact check (it was wrongly returning another class which encapsulated NewPrisonUserDto)